torch/nn/_intrinsic/quantized/modules/functional_relu.py

```diff
@@ -0,0 +1,17 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+from torch.nn._intrinsic import AddReLU as NNAddReLU
+
+class AddReLU(torch.nn.Module):
+    _FLOAT_MODULE = NNAddReLU
```

Comment:
I think `import torch.nn._intrinsic as nni` and use `nni.AddReLU` might be more clear. I'm changing to use this pattern in my pr as well.

Also, maybe a better name for this file? functional_relu seems to be confusing.

Reply:
Suggestions for the name of the file?

Reply:
Resolved in person -- decided to go with the current name for now